ch2.txt

```diff
@@ -30,6 +23,11 @@ CHAPTER 2. WAGING WAR
 
 16. Now in order to kill the enemy, our men must be roused to anger; that there may be advantage from defeating the enemy, they must have their rewards. 
 
+asdf
+asdf
+asdf
+adsf
+
```

Comment:
Suggestion text [0] is not a diff

```
arst
arst
arst
arst

```

[0]: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request